### DISCUSSION TITLE
Background Color For Header Card

### DISCUSSION THREAD

**Discussion by GokulMahendran** [h2oai/wave#666] [Q&A] [answered]
How to change background color for the header cards?

**Comment by mtanco** [comment, marked as answer]
Hi! And thank you for your interest in Wave!

The color of the header cannot be changed on its own, but Wave apps do have the concept of themes which allows for color changes throughout the app. Today there are two themes, light which is the default, and neon. You can change the theme using the `ui.meta_card(...theme='neon'...)` and see more here: https://h2oai.github.io/wave/docs/examples/meta-theme/

The ability to make custom themes will be available in a coming release (tentatively 0.15.0). Please checkout the demo video in this PR: https://github.com/h2oai/wave/pull/604
